app/src/pages/project/MetadataTableCell.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import { MetadataLabel } from "@phoenix/pages/project/MetadataLabel";
+import {
+  makeMetadataTooltipFilterCondition,
+  MetadataTooltip,
+} from "@phoenix/pages/project/MetadataTooltip";
```

Comment:
strange pattern of having the util in the component and exported but I guess I don't care a ton?